config/crd/incubator/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
+# Generated by scripts/generate-crd-kustomize.sh. DO NOT EDIT.
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  # TODO: make Renovate update this: https://github.com/Kong/kubernetes-ingress-controller/issues/6663
-  - https://github.com/kong/kubernetes-configuration/config/crd/ingress-controller-incubator?ref=v0.0.43
+  - https://github.com/kong/kubernetes-configuration/config/crd/ingress-controller-incubator?ref=v0.0.44
```

Comment:
Could you leave a comment everywhere where `v0.0.044` is hardcoded that it's updated automatically by `scripts/generate-crd-kustomize.sh`? Otherwise it may lead to confusion

Reply:
Added a comment in the generated `kustomization.yaml`. 👍